File: project3/Team3/projectBQ/src/main/java/bitc/full502/projectbq/dto/ItemSearchDto.java
package bitc.full502.projectbq.dto;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class ItemSearchDto {

//    비품 검색 (비품코드) DTO "20250917 완료"
    private String name;

    private Long categoryId;

    private String code;

    private Long warehouseId;

    private String keyword;

    private String manufacturer;

    private Long minPrice;

    private Long maxPrice;

    private LocalDateTime startDate;

    private LocalDateTime endDate;

    private String status;
}

File: project3/Team3/App_BQ/app/src/main/java/bitc/full502/app_bq/data/model/ItemSearchDto.kt
package bitc.full502.app_bq.data.model

import java.time.LocalDateTime

data class ItemSearchDto(
    val name: String? = null,
    val code: String? = null,
    val manufacturer: String? = null,
    val categoryId: Long? = null,
    val minPrice: Long? = null,   // 서버에서는 null로 보내도 됨
    val maxPrice: Long? = null,  // 서버에서는 null로 보내도 됨
)
